EncodingStreamingHighAvailability/media-services-high-availability-shared/Models/JobOutputStateChangeEventDataModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace media_services_high_availability_shared.Models
+{
+    using Microsoft.Azure.EventGrid.Models;
+    using Newtonsoft.Json;
+    public class JobOutputStateChangeEventDataModel : MediaJobOutputStateChangeEventData
+    {
+        public JobOutputStateChangeEventDataModel()
+        {
+            this.JobOutput = new JobOutputModel();
+        }
+
+        [JsonProperty(PropertyName = "output")]
+        public JobOutputModel JobOutput { get; set; }
```

Comment:
You shouldn't need this.  The output asset name should be included already.  Are you not seeing it?